### ISSUE TITLE
CodeActionProvider should support body expression  constructor with new action: Initialize body expression ctor from properties

### ISSUE THREAD

**Issue by bard83** [KreativJos/csharpextensions#122]
Body expression constructor is currently not supported as already mentioned in #56. Since all the others code actions have been removed, it does make sense to provide a new action to initialize the ctor - as body expression - from the properties.

New Action:
- Initialize body expression ctor from properties

possible body expression constructors:

```
public NewClass (int field1) => Field1 = field1;

public Field1 { get; set; }
```

```
public NewClass (int field1, int field2) => (Field1, Field2) = (field1, field2);

public Field1 { get; set; }
```

Left literals could be prefixed with `this.` or `_`.

**Comment by bard83** [comment]
The first case is covered by omnisharp yet. The second one could be more interesting.

**Comment by bard83** [comment]
closed by #125
